front/components/assistant/conversation/MessageItem.tsx

```diff
@@ -137,7 +137,9 @@ const MessageItem = React.forwardRef<HTMLDivElement, MessageItemProps>(
           <div
             key={`message-id-${sId}`}
             ref={ref}
-            className={index !== 0 ? "mt-6 w-full md:mt-10" : "w-full"}
+            className={
+              index !== 0 && !isLastMessage ? "my-6 w-full md:my-10" : "w-full"
```

Comment:
what do you think of reducing a tiny bit? 
```suggestion
              index !== 0 && !isLastMessage ? "my-6 w-full md:my-8" : "w-full"
```